packages/editor/src/components/post-saved-state/index.js

```diff
@@ -106,10 +103,6 @@ export default function PostSavedState( {
 		return null;
 	}
 
-	if ( isPublished || isScheduled ) {
-		return <PostSwitchToDraftButton />;
-	}
```

Comment:
Looks like this change is causing a small bug. Now for "published posts", the button is there and it says "Save draft", should we return `null` here instead?